### PR TITLE
Fabian/controller rating

### PR DESCRIPTION
**Pull request by fabianmueller7**

### CODE REVIEW

**Comment by DeNic0la** [comment]
@Laurin-dev and @fabianmueller7 This has Merge Conflicts resolve those and Request a Review if you want a Merge

File: rating/router.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    getRatingbyMovieId,
+    getRatingByUserId,
+    saveRating,
+} from "./controller.js";
+
+const router = Router();
+
+router.get("/averageRating/:movieId", getRatingbyMovieId);
+router.get("/getUserRating/:movieId", getRatingByUserId);
+router.get("/:movieId/:rating", saveRating);
```
Comment (DeNic0la) [inline rating/router.js:12]: No

**Review by DeNic0la — CHANGES_REQUESTED** [review]

File: rating/router.js (hunk 1 above)
Comment (fabianmueller7) [reply rating/router.js:12]: ?

File: rating/controller.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  let movieId = request.params.movieId;
+  console.log("Output" + movieId);
+  const data = await getRatingByMovieIdAndUserID(movieId, request.user.id);
+  response.send(data);
+}
+
+//expects movieid, rating
+export async function saveRating(request, response) {
+  console.log(request.params.movieId);
+  console.log(request.params.rating);
```
Comment (DeNic0la) [inline rating/controller.js]: you _could_ remove this

File: rating/router.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-    fabiansFunction
-} from './controller.js';
+    getRatingbyMovieId,
+    getRatingByUserId,
+    saveRating,
+} from "./controller.js";
 
 const router = Router();
 
-router.get('/:movieId/:rating', fabiansFunction);
```
Comment (DeNic0la) [inline rating/router.js:8]: This was correct and thats why i merged it in #7

File: rating/router.js [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+router.get("/averageRating/:movieId", getRatingbyMovieId);
+router.get("/getUserRating/:movieId", getRatingByUserId);
+router.get("/:movieId/:rating", saveRating);
```
Comment (DeNic0la) [inline rating/router.js:12]: i mean... it works in theory

**Review by DeNic0la — DISMISSED** [review]
Its far from Pretty but it looks about right

**Comment by DeNic0la** [comment]
@fabianmueller7 There are still some Issues with the Branch, it is out of Date

**Review by DeNic0la — APPROVED** [review]
We can do that

**Pull request merged**